packages/react-core/src/components/Modal/ModalBox.tsx

```diff
@@ -36,10 +42,15 @@ export const ModalBox: React.FunctionComponent<ModalBoxProps> = ({
     className={css(
       styles.modalBox,
       className,
+      position === 'top' && styles.modifiers.alignTop,
       variant === 'large' && styles.modifiers.lg,
       variant === 'small' && styles.modifiers.sm,
       variant === 'medium' && styles.modifiers.md
     )}
+    style={{
+      ...(distance && { '--pf-c-modal-box--m-align-top--spacer': distance }),
```

Comment:
you could also import `c_modal_box_m_align_top_spacer` from react-tokens and use 
`c_modal_box_m_align_top_spacer.name` as the key

Reply:
Syntax complains when I tried that, it doesn't like the `.name` and doesn't like `['name']` when it's used to index a property for an object.

Reply:
Wouldn't `[c_modal_box_m_align_top_spacer.name]: distance` work?